### ISSUE TITLE
Saves do not work between PCs

### ISSUE THREAD

**Issue by oddlyk1** [visualboyadvance-m/visualboyadvance-m#285]
### Expected behavior
Game saves on one PC and loads same file on another.

### Actual behavior
Second PC needs to import battery file to see the same saves. This causes the first PC to no longer be able to use the same file until importing again.

### Steps to reproduce the behavior
Using the same files, same saves, same config, via network share on 2 different PCs, try to load the same saves. Importing the battery save will fix the issue on one PC, but cause the same issue on the other.
Both PCs have the same access to the same files in the same virtual directory.

### ROM details
This appears to be the same on ALL roms, but I have tested on Pokemon [Red(GB)/Crystal(GBC)/Emerald(GBA)] and Fire Emblem(GBA).

### Build details
Build : Latest Stable 2.1.0
Operating System : Both PCs running Windows 10.

### NOTES:
I can't think of any reason VBA-M would load the save just fine on one PC and not the other. This may however be related to the 32/64 bit save bug (issue #283)

**Comment by retro-wertz** [comment]
battery related save files and save states are different things. its possible (at the moment) that save states between different OS and or architecture wont be compatible in vbam due to how the variables are treated but battery save file should have no difference and should be able to load between different version , OS or architecture, especially when using Import battery file works.
i dont even understand why you are loading from a network path...

**Comment by oddlyk1** [comment]
I have all of my emulators saved on my server so that they can be accessed on whichever PC I happen to be using. 
For example: Every time I load a game on my desktop, it automatically pick up on the battery file that was last used, however loading the same file, the same way, from the same path on my laptop it tries to create a NEW battery file to save to.

**Comment by retro-wertz** [comment]
whats your actual save path location? did you manually set Native Saves path? if this location is a network path e.g. "\\host\path" thingie, try to map your network save path to a local drive and see if this helps.

**Comment by oddlyk1** [comment]
That's actually how I already had it mapped. The set path for 'Native Saves' is "V:\[Emulators]\[GBA] VirtualBoyAdvance MERGE\N_SAVES" The 'V Drive' is mapped to the same share on both PCs.

**Comment by retro-wertz** [comment]
is this the actual location? **V:[Emulators][GBA]**? can you try without using "[" or "]" and just normal characters/numbers?

**Comment by oddlyk1** [comment]
Sorry, it somehow took out the backslashes.
`V:\ [Emulators]\[GBA] VirtualBoyAdvance MERGE\N_SAVES`
But i will try it using another folder that doesn't have the brackets.

**Comment by oddlyk1** [comment]
UPDATE: I moved the files to a more simplified directory `V:\VBA-M\` and set the Native Saves directory accordingly. This actaully made the problem WORSE somehow. Neither PC was auto-loading the battery file. HOWEVER... I deleted my config file (vbam.ini) and started from scratch. Both PCs now seem to load the batter fine. It's frustrating that I have no idea what caused the issue... But I'm glad it works now.
